cmake/modules/ZyppCommon.cmake

```diff
@@ -60,7 +60,7 @@ ENDIF( NOT DEFINED DOC_INSTALL_DIR )
 
 SET( ZYPPCOMMON_CXX_STANDARD 17 )
 #SET (CMAKE_INCLUDE_DIRECTORIES_BEFORE ON)
-INCLUDE_DIRECTORIES( ${CMAKE_CURRENT_SOURCE_DIR} ${PROJECT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR} SYSTEM )
+#INCLUDE_DIRECTORIES( ${CMAKE_CURRENT_SOURCE_DIR} ${PROJECT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR} SYSTEM )
```

Comment:
Any reason to keep this line in the comment? Such commented code has the tendency to stick for years and nobody feels the authority to remove them.